air_to_breath_resources/utils/sensor.py

```diff
@@ -135,17 +138,17 @@ def set_value(self, sent_value: int):
         else:
             data = sent_value
 
-        data_encoded = str(data / 1000).encode()
+        data_encoded = str(data).encode()

```

Comment:
why did you removed it?
if you remove it than its the same as socket sensor, and you don't need to override it

Reply:
They changed the sensor